### PR TITLE
Improve Travis

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by bobsummerwill** [comment]
I'm guessing this is still WIP, eh?

**Comment by bobsummerwill** [comment]
Are you doing anything with this, @chfast?

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: cpp
-cache: ccache
+cache:
+    - ccache
+    - directories:
+        - ~/.ethash
```
Comment (chfast) [inline .travis.yml:5]: Caching DAGs seems good idea.

**Comment by chfast** [comment]
This can be closed. I tried to workaround broken llvm package of Ubuntu.

The only valuable part is Travis config to cache ethash DAG.

**Comment by bobsummerwill** [comment]
OK - why don't you make a new PR for the DAG caching against cpp-ethereum and then we're done with this?  Thanks!

**Pull request closed without merging**